lib/push.js

```diff
@@ -489,6 +489,7 @@ module.exports = function (log, db, config) {
                 device.pushCallback = ''
                 device.pushPublicKey = ''
                 device.pushAuthKey = ''
+                device.pushEndpointExpired = true
```

Comment:
Since we're now setting an explicit flag, I wonder if we should leave the `pushCallback` etc intact here for e.g. debugging purposes.